site/en/tutorials/generative/deepdream.ipynb

```diff
@@ -82,49 +111,47 @@
         "\n",
         "Let's demonstrate how you can make a neural network \"dream\" and enhance the surreal patterns it sees in an image.\n",
         "\n",
-        "![Dogception](images/dogception.png)"
+        "![Dogception](https://github.com/sayakpaul/docs/blob/master/site/en/tutorials/generative/images/dogception.png?raw=1)"
```

Comment:
Keep image same

Reply:
Sure. Thanks!

Reply:
Hi @lamberta I also changed the bullet points for the following section in [this tutorial](https://www.tensorflow.org/tutorials/keras/save_and_load):

![Screen Shot 2019-10-27 at 2 50 36 PM](https://user-images.githubusercontent.com/22957388/67632546-d3ae2b80-f8ca-11e9-861d-dbe1df9a3ee4.png)

Although "*" allows us to create bullets in Markdown, for some weird reasons it is rendering properly (which you can see above). I replaced the Asterisk with "-".

Reply:
@lamberta any updates on this?

Reply:
tensorflow.org's .markdown parser doesn't recognize lists unless they have a blank line before them.

@lamberta:

Did you mean switch to tensorflow/docs or switch back to a relative link?

Reply:
@MarkDaoust I meant keep the image path as-is: relative and not to a personal repo.

Reply:
> tensorflow.org's .markdown parser doesn't recognize lists unless they have a blank line before them.

@MarkDaoust thanks for passing that along :)

Reply:
@yashk2810 could you provide your review on this?